diff --git a/sir/indexing.py b/sir/indexing.py
index e38e61b..efea1cd 100644
--- a/sir/indexing.py
+++ b/sir/indexing.py
@@ -335,6 +335,7 @@ def send_data_to_solr(solr_connection, data):
     with sentry_sdk.new_scope() as scope:
         scope.set_extra("data", data)
         try:
+            logger.debug("Sending data to Solr: %s", data)
             solr_connection.add(data)
             logger.debug("Done sending data to Solr")
         except Exception as e:
